index.css

```diff
@@ -0,0 +1,547 @@
+@media only screen and (max-width: 786px) {
+  .body {
+    width: 50px;
+  }
+  .main-content1 {
+    width: 50px;
+  }
+  .main-content1-wrapper {
+    width: 50px;
+  }
+}
+
+#header {
+  border-bottom: 6px double lightgray;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 325px;
+  margin-right: 325px;
+}
+
+#header h1 {
+  font-size: xx-large;
+}
+
+.header-link {
+  color: tomato;
+  text-decoration: none;
+}
+
+.header-link:hover {
+  background-color: lightgray;
+}
+
+#header h6 {
+  text-align: center;
+}
+
+#header,
+h1,
+h6 {
+  text-align: center;
+}
+
+h1 {
+  color: tomato;
+}
+
+.content {
+  display: flex;
+  justify-content: center;
+}
+
+#homeimgs {
+  display: flex;
+  justify-content: center;
+}
+
+.img1wrapper {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  padding: 10px;
+  align-items: center;
+}
+
+.img2wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  align-items: center;
+}
+
+.img3wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  align-items: center;
+}
+
+.img1 {
+  width: 200px;
+  height: 200px;
+  display: flex;
+  align-items: center;
+}
+
+.img2 {
+  width: 200px;
+  height: 200px;
+}
+
+.img3 {
+  width: 200px;
+  height: 200px;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+}
+
+.button1 {
+  background-color: orange;
+  width: 150px;
+  color: white;
+}
+
+.button1:hover {
+  background-color: green;
+}
+
+/* Next section */
```

Comment:
Get more descriptive with your section names, is this a different page or is this the body of the home page? If you label things more clearly it'll be easier to read through